.gitignore

```diff
@@ -1,3 +1,6 @@
 /target
 /Cargo.lock
-/.embuild
\ No newline at end of file
+/.embuild
+/.devcontainer
```

Comment:
You ignore the devcontainer folder here. But this PR also commits devcontainer definitions. I think you might want to remove the devcontainer files from this PR and keep it ignored?